stan/math/fwd/scal/fun/inv_logit.hpp

```diff
@@ -4,6 +4,7 @@
 #include <stan/math/fwd/meta.hpp>
 #include <stan/math/fwd/core.hpp>
 #include <stan/math/prim/scal/fun/inv_logit.hpp>
+#include <cmath>
```

Comment:
[question]
Any idea why cpplint didn't catch this?  I thought they were supposed to catch problems with include-what-you-use from the std:: lib.  Anyway, thanks for finding and fixing all this!

Reply:
I was amazed also, so dont have an answer. I made a simple Python script and just ran it through  `stan/math` and fixed the files the script flagged.